README.md

```diff
@@ -13,6 +13,7 @@ for how to install Foreman plugins
 
 | Foreman Version | Katello Version | Plugin Version |
 | --------------- | --------------- | -------------- |
+| 2.3             | 3.18            | ~> 1.8.9       |
```

Comment:
I guess, this is in preparation for the next version!?

Reply:
Yes. Is it better to add this with the version bump?

Reply:
Maybe, but it is fine here as well.